### DISCUSSION TITLE
Parsed mesh.json files

### DISCUSSION THREAD

**Discussion by JonMike8** [stevensblueprint/nycmesh-node-visualizer#19] [General]
# Files for Issue #17
[access_points.csv](https://github.com/stevensblueprint/nycmesh-node-visualizer/files/12811613/access_points.csv)
[access_points.txt](https://github.com/stevensblueprint/nycmesh-node-visualizer/files/12811630/access_points.txt)

I wasn't able to upload a .json file to a discussion so I just copied it to a .txt file. The .json file is in the discord. Feel free to let me know if there's anything I need to change.

**Comment by miguel-merlin** [comment]
#17
